ci/install_libevent.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+apt-get install --no-install-recommends --no-install-suggests -y \
+                libevent-dev
```

Comment:
Can this just be merged into setup_ci_environment?

Reply:
It's not needed for all the test targets, so separating it out makes builds some of the targets faster to build.

It doesn't matter much for linux where binary packages are used, but windows builds libevent from source.

Reply:
Fair enough.